source/patterns/@aws-solutions-constructs/core/lib/elasticsearch-helper.ts

```diff
@@ -257,7 +257,11 @@ export function GetSubnetsAndSecurityGroups(lambdaFunction: lambda.Function, vpc
     }
```

Comment:
lines 238-254 are really unintelligible. Lots of 'inside baseball' elasticsearch domain terms. I can't figure out what it is doing, so I can't suggest improvements yet.